src/components/Request/NewFormSection/NewForm.Done.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Divider, Text, Title2 } from "@fluentui/react-components";
+import { useFormContext } from "react-hook-form";
+
+const Done = ({ gotoStep }: { gotoStep: (step: string) => void }) => {
+  const form = useFormContext();
+
+  const values = form.getValues();
+  console.log("Values:", values);
+
+  const HiringInfo =
+    values.advertisementLength === "" || values.methods.length === 0;
+
+  const JobBoard: boolean =
+    values.methods.includes("lcmc") && values.closeDateLCMC === undefined;
+
+  const JOA: boolean =
+    values.methods.includes("joa") &&
+    (!values.closeDateJOA ||
+      !values.organizationalPOC ||
+      !values.issueTo ||
+      !values.fullPartTime ||
+      !values.salaryLow ||
+      !values.salaryHigh ||
+      !values.telework ||
+      !values.remote ||
+      !values.pcs ||
+      !values.joaQualifications ||
+      !values.joaIdealCandidate);
+
+  const LinkedInPost: boolean =
+    values.methods.includes("linkedinPost") &&
+    (!values.temporary ||
+      !values.salaryLow ||
+      !values.salaryHigh ||
+      !(values.temporary === "Full" ? true : values.nte) ||
```

Comment:
String should be "Full-Time"

```suggestion
      !(values.temporary === "Full-Time" ? true : values.nte) ||
```